readme.md

```diff
@@ -34,6 +34,7 @@ __Build what matters__
 __Do *big* things in small steps__
   
   * Breaking down big features in small steps doesn’t mean we work only on small incremental improvement. A longer term vision or objective help us achieve great things without sacrificing short term value.
+  * Make use of feature flags to allow you release early and often to test assumptions, especially in production.
```

Comment:
I think this is a little too implementation specific rather than organisational

Reply:
See my suggestion. I think that this _is_ a principle, but could perhaps be reworded a little to express it as such.

Reply:
I think the language should be expressed in a more generic manner that doesn't include terms like "conditionals" or "flags" or "country.

Reply:
Something like `Release work in progress features using tools to allow internal users to test them, ensuring a degree of production-readiness is constantly kept`

Reply:
OK, for point. Version 2.1:

> Feature-specific toggles are a good way to release early and often. By coupling each well-defined feature with its own toggle, we can better control when and where to activate it.

Reply:
👍

Reply:
Nice, I like the toms suggestion, ill change it to that. Appreciate its implementation specific in a way but I think its import to be explicit, and allow the principles to be educational in this example too

Reply:
After each principle I'd like to add examples. You can add a section that says: "Example: * We use feature toggles to release to production early and often. We currently use Launch Darkly to manager our features".

Reply:
@soulnafein ive amended the original text to match Tom's now, so this is out of date, ill address your comment on the new text